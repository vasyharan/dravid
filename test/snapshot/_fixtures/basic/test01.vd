def foo(a, b, c) {
   foo = 10 + 10 * 10
}
