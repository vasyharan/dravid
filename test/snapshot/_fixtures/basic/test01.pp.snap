(fn (proto foo
           ((param var a)
            (param var b)
            (param var c)))
    ((asgn 
      (id foo)
      (*
       (+
        (int 10)
        (int 10))
       (int 10)))))
